app/api/routes.py

```diff
@@ -243,8 +243,19 @@ def search_results():
     page = request.args.get('page', 0, int)
     page_size = request.args.get('page_size', Config.RESOURCE_PAGINATOR.per_page, int)
 
+    # Fetch the filter params from the url, if they were provided.
+    paid = request.args.get('paid')
+    filters = ''
+
+    # Filter on paid
+    if isinstance(paid, str):
+        # algolia filters boolean attributes with either 0 or 1
+        algolia_bool = 1 if paid.lower() == "true" else 0
```

Comment:
I'm on the fence about this. I feel like if it's not explicitly `"true"` or `"false"` it should return all resources. I might be convinced why this is a better way though, can you discuss why you chose to have it default to false rather than all?

Reply:
I mainly did it this way because the /resource  endpoint filtering also defaulted to false, that way we could expect similar behavior in filtering here as well

Reply:
Oh.. we should probably have the `/resource` filtering default to `all` if an invalid `paid` attribute is given. Can you open up an issue for that? Seems like a beginner friendly one we can allow another contributor to get in on if that's ok

Reply:
Sure thing, do you want me to default to all as well or leave it also with that issue?

Reply:
Yeah, go ahead and change this PR to default to all if `paid` is not a boolean. Thanks